src/app/app.component.html

```diff
@@ -1,2 +1,9 @@
-<app-nav-bar></app-nav-bar>
-<app-mood-ometer></app-mood-ometer>
+<nav class="navbar-nav fixed-top navbar-expand-md navbar-light bg-light Navigation">
+  <div id="logo"></div>
+  <nav style="padding-left: 10px; padding-top: 20px">
+    <a routerLink="/mood">MoodOMeter</a>
```

Comment:
please use "moodometer", it's easier to follow what it is

Reply:
will be fixed on next push